### PR TITLE
user guide for 1.0.7 pre-release features

### LINKED ISSUE #32: User guide for 1.0.7
For our new pre-release we need to write a user guide.

### PR DESCRIPTION
**Pull request by pj-spoelders**
Closes #32

### CODE REVIEW

File: doc/user_guide/user_guide_107.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
+# Exasol Power BI Connector 1.0.7 Release - User Guide
```
Comment (jakobbraun) [inline doc/user_guide/user_guide_107.md:1]: why do you call it `107.md`. It versioned it git anyway.

File: doc/user_guide/user_guide_107.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+- DSN (Data Source Name) Support
+
+- Limited Kerberos support
+
+## Installing the connector:
+
+### Download the .mez
+
+Copy the ''Exasol.mez' file available on : [Releases · exasol/powerbi-exasol · GitHub](https://github.com/exasol/powerbi-exasol/releases)
```
Comment (jakobbraun) [inline doc/user_guide/user_guide_107.md:17]:
```suggestion
Download the 'Exasol.mez' file available on : [Releases · exasol/powerbi-exasol · GitHub](https://github.com/exasol/powerbi-exasol/releases)
```

File: doc/user_guide/user_guide_107.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
+# Exasol Power BI Connector 1.0.7 Release - User Guide
+
+## New features
```
Comment (jakobbraun) [inline doc/user_guide/user_guide_107.md:3]: Usually we don't include the new features here but in the changes_x.x.x.mde file.

File: doc/user_guide/user_guide_107.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+- Custom Query support
```
Comment (jakobbraun) [inline doc/user_guide/user_guide_107.md:5]: But the connector is also bundeled with power bi right?
Maybe you can tell here somewhere that it's just for the advanced users, and defualt users can pic the built-in connector

File: doc/user_guide/user_guide_107.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+
+Create the directory path if it doesn't exist.
+
+### Change the PowerBI Desktop settings
+
+Go to options in PowerBI:
+
+File > Options and settings > Options
+
+![](./img/2021-07-16-13-42-57-image.png)
```
Comment (jakobbraun) [inline doc/user_guide/user_guide_107.md:37]: :+1 nice to have images here

Comment (jakobbraun) [reply doc/user_guide/user_guide_107.md:37]: Could you rename them, with speaking names.

Reasons:
* Helps with screen readers
* Helps for search engine optimization

File: doc/user_guide/user_guide_107.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+
+Navigate to the security tab.
+
+Change the data extensions setting to allow the custom connector to load.
+
+![](./img/2021-07-16-13-28-43-image.png)
+
+### Shut down and restart Power BI Desktop
```
Comment (jakobbraun) [inline doc/user_guide/user_guide_107.md:45]:
```suggestion
### Restart Power BI Desktop
```

File: doc/user_guide/user_guide_107.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+
+- Custom Query support
+
+- ODBC Driver Logging
+
+- DSN (Data Source Name) Support
+
+- Limited Kerberos support
+
+## Installing the connector:
```
Comment (jakobbraun) [inline doc/user_guide/user_guide_107.md:13]: Use assotiated press headline style: https://headlinecapitalization.com/

File: doc/user_guide/user_guide_107.md [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+
+
+
+Important note: You can only use a custom query in "import" mode. You can also use DirectQuery but won't be able to use the results in any visuals. 
+
+(You can alter the mode afterwards in the model screen in case you accidentally picked the unsupported option.)
+
+# Limited Kerberos support
+
+We also support an 'easy' kerberos authentication mode where the KERBEROSSERVICENAME will be exasol/<dnsrecord> if you choose the kerberos authentication option.
```
Comment (jakobbraun) [inline doc/user_guide/user_guide_107.md:141]: ``
```suggestion
We also support an 'easy' kerberos authentication mode where the `KERBEROSSERVICENAME` will be `exasol/<dnsrecord>` if you choose the kerberos authentication option.
```

File: doc/user_guide/user_guide_107.md [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+
+Select your data source, click the "advanced" tab.
+
+![](./img/2021-07-16-15-55-16-image.png)
+
+You can easily turn logging on and off as well as select a desired log mode using the "Log file and "Log mode" fields.
+
+Another option to do this is to manually add the relevant connection string keys to the connection string or to add them to Additional Connection string parameters under 'Advanced'.
+
+In this case that would be: LOGMODE=DEBUGCOMM;EXALOGFILE=C:\tmp\pbiblog.txt;
```
Comment (jakobbraun) [inline doc/user_guide/user_guide_107.md:109]:
```suggestion
In this case that would be: `LOGMODE=DEBUGCOMM;EXALOGFILE=C:\tmp\pbiblog.txt`;
```

Also add to other inline code snippets

**Review by jakobbraun — CHANGES_REQUESTED** [review]

File: doc/user_guide/user_guide_107.md (hunk 4 above)
Comment (pj-spoelders) [reply doc/user_guide/user_guide_107.md:5]: it's for limited use

**Pull request merged**